shell/package.json

```diff
@@ -105,6 +109,7 @@
     "marked": "4.0.17",
     "nodemon": "2.0.22",
     "nyc": "15.1.0",
+    "node-polyfill-webpack-plugin": "^3.0.0",
```

Comment:
You are right, I should have been pinning the versions as well. Would you mind to do that? 😬